services/ownerService.js

```diff
@@ -41,13 +42,18 @@ exports.addHotel = async (data) => {
   return hotel
 }
 
-exports.updateHotel = async (id, data) => {
-  const hotelUpdate = await Hotel.findByIdAndUpdate(id, data)
+exports.updateHotel = async (id, data, req) => {
```

Comment:
Do these params matches arguments that are passed when we call this function? I mean here we have req and where we call this function we pass req.user._id